src/js/jsx/sections/transform/Size.jsx

```diff
@@ -42,7 +43,36 @@ define(function (require, exports, module) {
     var Size = React.createClass({
         mixins: [FluxMixin],
 
-        shouldComponentUpdate: function (nextProps) {
+        /**
+         * Variables for storing inital scrub value to determine correct offset.
+         * @private 
+         * @type {number}
+         */
+        _initialScrubW: null,
```

Comment:
I'm not sure I see the value of having this exist as a private variable outside of `state`.  Isn't it just "the uniform value of the width"?  (If the widths are not uniform, then scrubbing is essentially disabled)